exchanges/poloniex/poloniex_websocket_trade.go

```diff
@@ -0,0 +1,93 @@
+package poloniex
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/encoding/json"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+)
+
+// WsCreateOrder create an order for an account.
+func (e *Exchange) WsCreateOrder(ctx context.Context, arg *PlaceOrderRequest) (*OrderIDResponse, error) {
+	if err := validateOrderRequest(arg); err != nil {
+		return nil, err
+	}
+	var resp []*OrderIDResponse
+	if err := e.SendWebsocketRequest(ctx, "createOrder", arg, &resp); err != nil {
+		return nil, fmt.Errorf("%w %w", order.ErrPlaceFailed, err)
+	}
+	if len(resp) != 1 {
+		return nil, common.ErrInvalidResponse
+	} else if resp[0].Code != 0 && resp[0].Code != 200 {
```

Comment:
🚧 Change request:
I've been accepting/ignoring this `Code == 0` is alright paradigm everywhere, and just realised to question it.
Why are you saying that a code of 0 is okay?
The docs don't support that.

Reply:
But @samuael ... This is a websocket request in a file called websocket_trade.go ?
You're using reasoning that REST v1 and v2 requests should affect the code written specifically for WS requests.
That's not reasonable at all.
I'll stick to this request and highlight where I want changes.